src/main.rs

```diff
@@ -94,6 +100,12 @@ fn main() {
                 .help("Control serial port: off|tty|file=/path/to/a/file")
                 .default_value("tty"),
         )
+        .arg(
+            Arg::with_name("console")
+                .long("console")
+                .help("Control (virtio) conslole: off|hvc|file=/path/to/a/file")
```

Comment:
It should be tty as that refers to the *host* tty.

Reply:
Done.